src/compas/datastructures/network/operations/join.py

```diff
@@ -54,8 +54,8 @@ def network_polylines(network, splits = []):
     ----------
     network : Network
         A network.
-    splits : list
-        List of point coordinates for optional splits.
+    splits : list, None
```

Comment:
proper way to write this would be
```python
splits : list, optional
    List of point coordinates for optional splits.
    Default is ``None``
```